popup.css

```diff
@@ -0,0 +1,57 @@
+.overlay {
+  width: 100%;
+  height: 100%;
+  background-color: #c1c7d0;
+  opacity: 0.9;
+  position: fixed;
+  top: 0;
+  left: 0;
+  padding: 16px;
+  overflow: auto;
+  display: none;
+}
+
+.popup-card {
+  background-color: #fff;
+  border-radius: 10px;
+  height: 100%;
+  padding: 12px;
+}
+
+.popup-close-btn {
+  color: #67798e;
+  font-size: 24px;
+  font-weight: 900;
+  position: absolute;
+  top: 35px;
+  right: 32px;
+}
```

Comment:
- [ ] I would kindly suggest we stick to the Figma design as much as possible, currently the `desktop` and `mobile` versions of your pop-up modal look quite different from what we have in the Figma template, the alignment, and the positioning.

Your design 👇 


| mobile screen  | desktop screen |
| ------------- | ------------- |
|  ![sam1](https://user-images.githubusercontent.com/78634841/209415203-ac6ba56c-edea-4477-bc86-d6f7576cbfa2.png) | ![sam](https://user-images.githubusercontent.com/78634841/209415219-aa8cee8b-5544-4f96-acd7-4f1292e02d67.png)  |


Figma design 👇 

| mobile screen | desktop screen |
| --- | --- |
| ![real-pop](https://user-images.githubusercontent.com/78634841/209415271-c359ff45-a030-4b00-a638-ecfe3673a9d5.png) | ![pp1-real](https://user-images.githubusercontent.com/78634841/209415443-e406dc8b-636d-419f-bb35-6cb3f0909bd6.png) |


  - kindly check the text alignment, the image size, the background content blur, and the button for the desktop and mobile versions.

Thank you.